test/client/bespoke-client-test.ts

```diff
@@ -37,11 +37,12 @@ describe("BespokeClient", function() {
         });
 
         it("Connects to something other than localhost", function(done) {
-            let client = new BespokeClient("JPK", "proxy.bespoken.tools", 5000, "0.0.0.0", 9001);
+            this.timeout(5000);
+            let client = new BespokeClient("JPK", "proxy.bespoken.tools", 5000, "0.0.0.0", 443);
             client.onConnect = function (error: any) {
```

Comment:
Might add a console log on this error, in case for some reason one is being received